app/models/food.rb

```diff
@@ -1,6 +1,13 @@
 class Food < ApplicationRecord
-  belongs_to :user
-  validates :name, presence: true
-  validates :price, presence: true, numericality: { greater_than_or_equal_to: 0 }
-  validates :quantity, presence: true, numericality: { greater_than_or_equal_to: 0 }
+  after_create :save_to_database
+  # belongs_to :user
+  # validates :name, presence: true
+  # validates :price, presence: true, numericality: { greater_than_or_equal_to: 0 }
+  # validates :quantity, presence: true, numericality: { greater_than_or_equal_to: 0 }
```

Comment:
Since food belongs to the user, and validations are needed for model secure operations, it'd better to uncomment those validations and associations.